src/test/java/com/wikia/webdriver/common/contentpatterns/AdSlot.java

```diff
@@ -0,0 +1,29 @@
+package com.wikia.webdriver.common.contentpatterns;
+
+import lombok.Getter;
+
+public enum AdSlot {
+  TOP_LEADERBOARD("top_leaderboard","div[id$='top_leaderboard_0__container__']"),
```

Comment:
We need to think about the selectors used here:
1) if they're all consistent it's easy to understand it
2) `div[id$='top_leaderboard_0__container__']` is nice because it checks if ad server really responded but it's getting complicated once we change ad unit to MEGA...

Reply:
Discussed offline.